courseworks/week2/Shiqinrui_Xu/2D_brown_motion.jl

```diff
@@ -0,0 +1,37 @@
+using Random, Plots
+
+function simulate_brownian_motion_2d(n_steps)
+    # Set initial position to the origin
+    pos = [0.0, 0.0]
+    
+    # Initialize an array to store the particle trajectory
+    traj = zeros(n_steps+1, 2)
+    traj[1, :] = pos
+    
+    # Set the standard deviation of the step size
+    step_size = 0.1
+    
+    # Loop over the specified number of steps
+    for i in 1:n_steps
+        # Generate a random step in x and y directions
+        dx, dy = step_size * randn(2)
```

Comment:
notice that $dx$, $dy$ should be generated by a 2D gaussian distribution, but not an evenly distribution

Reply:
Your anwers are correct, but you need to solve the conflict so that I can merge your branch